profiler/src/main/java/com/splunk/opentelemetry/profiler/JFR.java

```diff
@@ -29,7 +29,12 @@
   public static final JFR instance = new JFR();
 
   public boolean isAvailable() {
-    return FlightRecorder.isAvailable();
+    try {
+      JFR.class.getClassLoader().loadClass("jdk.jfr.FlightRecorder");
```

Comment:
It might be better to do class available check in static initializer and keep the result in a field so you won't accidentally do it multiple times.

Reply:
There are only 2 calls and they're early in the lifecycle, but it's still a good idea...so I did that.  Thanks! 👍🏻